### ISSUE TITLE
Test exit with failure when using @exponent/vector-icons

### ISSUE THREAD

**Issue by ecstasy2** [expo/jest-exponent#7]
Here is my component

```
import React, { PropTypes } from 'react'
import {
  Text,
  View,
  StyleSheet,
  TouchableOpacity,
} from 'react-native'

import {Ionicons} from '@exponent/vector-icons';

class DigitKeypad extends React.Component {

  render () {
    return (<Ionicons
      name='ios-backspace-outline'
      size={32}
      color='white'
      style={{textAlign: 'center', left: -5}}
    />)
  }
}
````

The corresponding test is very straightforward

```
import 'react-native';
import React from 'react';

import MyComponent from './VerifyConfirmCode';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(
    <MyComponent />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
```

I obtain the following error when I run jest:

```
TypeError: Cannot read property 'Object.<anonymous>' of null
    at Runtime._execModule (/Users/diallo/git/smart/magic-app/node_modules/jest-runtime/build/index.js:444:56)
    at Runtime.requireModule (/Users/diallo/git/smart/magic-app/node_modules/jest-runtime/build/index.js:295:14)
    at Runtime.requireModuleOrMock (/Users/diallo/git/smart/magic-app/node_modules/jest-runtime/build/index.js:365:19)
    at reportException (/Users/diallo/git/smart/magic-app/node_modules/react-native/Libraries/Core/ExceptionsManager.js:21:21)
    at CustomConsole.reactConsoleErrorHandler [as error] (/Users/diallo/git/smart/magic-app/node_modules/react-native/Libraries/Core/ExceptionsManager.js:89:1)
    at /Users/diallo/git/smart/magic-app/node_modules/core-js/modules/es6.promise.js:117:9
    at perform (/Users/diallo/git/smart/magic-app/node_modules/core-js/modules/es6.promise.js:59:1)
    at Immediate._onImmediate (/Users/diallo/git/smart/magic-app/node_modules/core-js/modules/es6.promise.js:111:8)
    at processImmediate [as _immediateCallback] (timers.js:383:17)
```

**Comment by ecstasy2** [comment]
I was able to resolve this issue by mocking `exponent/vector-icons`. It seems like it is due to requiring `.ttf` files which are not supported for some reason.
Here is the tweak I am using:

```javascript
// NOTE: This might change in the future
// This mock @exponent/vector-icons because jest don't support
// loading ttf file
jest.mock('@exponent/vector-icons', () => {
  var MyReact = require('react')
  var IonIconMock = MyReact.createClass({
    render: function() {
      const props = this.props
      const state = this.state
      return MyReact.createElement('IonIcon', props, state)
    }
  });

  return {
    Ionicons: IonIconMock
  };
});
```

Any idea on how to get around this without mock?

**Comment by dikaiosune** [comment]
@brentvatne

**Comment by brentvatne** [comment]
Hi there! The issue was related to not having robust enough mocks for ExponentFileSystem and ExponentFontLoaded, this has been resolved in 0.3.1 :) Thanks for reporting the issue!
